pkg/injector/annotations/annotations.go

```diff
@@ -47,6 +47,9 @@ const (
 	KeyReadinessProbePeriodSeconds      = "dapr.io/sidecar-readiness-probe-period-seconds"
 	KeyReadinessProbeThreshold          = "dapr.io/sidecar-readiness-probe-threshold"
 	KeySidecarImage                     = "dapr.io/sidecar-image"
+	KeySidecarProfile                   = "dapr.io/sidecar-profile"
```

Comment:
Can you be more specific on what kind of profile this is? "sidecar-profile" is not immediately clear without context

Reply:
Yeah, removed this annotation as it might be confusing.